core/modules/replica/Performance.h

```diff
@@ -22,31 +22,24 @@
 #define LSST_QSERV_REPLICA_PERFORMANCE_H
 
 /**
- * This header declares vaious utilities (classes) which are used
+ * This header declares vairous utilities (classes) which are used
  * for timing and performance measurements of the Replication system's
  * operations.
  */
 
 // System headers
 #include <iostream>
 
-// Forward declarations
-
-namespace lsst {
-namespace qserv {
-namespace proto {
-
-/// The protocol class
-class ReplicationPerformance;
-
-}}} // namespace lsst::qserv::proto
-
 // This header declarations
 
 namespace lsst {
 namespace qserv {
 namespace replica {
 
+// Forward declarations
+
+class ProtocolPerformance;
```

Comment:
the way the forward declarations were is consistent with forward declarations in qserv.

Reply:
If someone told me what is the "qserv way" of making forward declarations then I would be happy to follow the lead 😸 When I'm looking at the headers in Qserv modules with:
```
egrep -B 5 -A 5 '[Ff]orward' core/modules/*/*.h
```
Then I'm seeing a spectrum of patterns, including (empty lines inside namespaces and no indentations):
```
core/modules/wbase/MsgProcessor.h:// Forward declarations
core/modules/wbase/MsgProcessor.h-namespace lsst {
core/modules/wbase/MsgProcessor.h-namespace qserv {
core/modules/wbase/MsgProcessor.h-namespace wbase {
core/modules/wbase/MsgProcessor.h-
core/modules/wbase/MsgProcessor.h-class Task;
core/modules/wbase/MsgProcessor.h-class WorkerCommand;
core/modules/wbase/MsgProcessor.h-
core/modules/wbase/MsgProcessor.h:}}} // End of forward declarations
core/modules/wbase/MsgProcessor.h-
```
And this:
```
core/modules/xrdsvc/SsiService.h:// Forward declarations
core/modules/xrdsvc/SsiService.h-class XrdSsiLogger;
core/modules/xrdsvc/SsiService.h-
core/modules/xrdsvc/SsiService.h-namespace lsst {
core/modules/xrdsvc/SsiService.h-namespace qserv {
core/modules/xrdsvc/SsiService.h-namespace wcontrol {
core/modules/xrdsvc/SsiService.h-  class Foreman;
core/modules/xrdsvc/SsiService.h-}
core/modules/xrdsvc/SsiService.h-namespace wpublish {
core/modules/xrdsvc/SsiService.h-  class ChunkInventory;
core/modules/xrdsvc/SsiService.h:}}} // End of forward declarations
```
core/modules/ccontrol/UserQuery.h:// Forward decl
core/modules/ccontrol/UserQuery.h-namespace lsst {
core/modules/ccontrol/UserQuery.h-namespace qserv {
core/modules/ccontrol/UserQuery.h-namespace qdisp {
core/modules/ccontrol/UserQuery.h-class MessageStore;
core/modules/ccontrol/UserQuery.h-}}}
```
And this:
```
core/modules/parser/SelectParser.h:// Forward declarations
core/modules/parser/SelectParser.h-namespace lsst {
core/modules/parser/SelectParser.h-namespace qserv {
core/modules/parser/SelectParser.h-namespace parser {
core/modules/parser/SelectParser.h-    class AntlrParser; // Internally-defined in SelectParser.cc
core/modules/parser/SelectParser.h-}
core/modules/parser/SelectParser.h-namespace query {
core/modules/parser/SelectParser.h-    class SelectStmt;
core/modules/parser/SelectParser.h:}}} // End of forward declarations
```

Reply:
And, unfortunately, there s noting on this subject in the Style Guide:
https://developer.lsst.io/cpp/style.html

Reply:
Sorry, my point was the forward declarations are consistently in their own block, not in the namespacing scope of the class being declared.

Reply:
Got it! I will refactor my code to follow this convention.